cmd/aws-lambda-rie/handlers.go

```diff
@@ -94,11 +94,12 @@ func InvokeHandler(w http.ResponseWriter, r *http.Request, sandbox Sandbox) {
 
 	invokeStart := time.Now()
 	invokePayload := &interop.Invoke{
-		ID:              uuid.New().String(),
-		TraceID:         r.Header.Get("X-Amzn-Trace-Id"),
-		LambdaSegmentID: r.Header.Get("X-Amzn-Segment-Id"),
-		Payload:         bodyBytes,
-		CorrelationID:   "invokeCorrelationID",
+		ID:                 uuid.New().String(),
+		InvokedFunctionArn: fmt.Sprintf("arn:aws:lambda:us-east-1:012345678912:function:%s", GetenvWithDefault("AWS_LAMBDA_FUNCTION_NAME", "test_function")),
```

Comment:
nit: show we move the defaults such as test_function, default memory of 3008 etc to a constants.go ?

Reply:
Not sure if that could be an issue or not, but are we sure this account number doesn't exist?
Do we have example/dummy account numbers for this?

Reply:
@mgrandis This is typically the account used as "fake" from what I have seen. I have no idea if it is a real account but we need something that looks like an account id. Do you have a better suggestion?

@sriram-mv Given this is likely not to change much at all. I am going to leave as is for now. Will keep an eye on this and refactor if this comes up again.